CITi-UFPE/CITiMarco
# update(feat/Input-screen): creating input page base

## What I've done
- Created the Input page base 

## How to test

`1. yarn `
`2. yarn build `
`3. yarn start`

See if the screen looks like the mockup in different sizes.
